### ISSUE TITLE
False positive on scipy.io.loadmat

### ISSUE THREAD

**Issue by PowerSnail** [PyCQA/pylint#1994]
### Steps to reproduce
1. import `scipy.io`
2. use `scipy.io.loadmat`
3. run pylint on the file

Minimal python file to reproduce this:

```python
"""test.py
"""
import scipy.io


if __name__ == '__main__':
    MAT = scipy.io.loadmat('unnamed.mat')
```

```bash
> pylint test.py
No config file found, using default configuration
************* Module test
E:  7,10: Module 'io' has no 'loadmat' member (no-member)  
```

### Current behavior

Report a "no-member" error 

### Expected behavior

No report

### pylint --version output

```
No config file found, using default configuration
pylint 1.8.2,
astroid 1.6.1
Python 3.6.4 (v3.6.4:d48eceb, Dec 19 2017, 06:54:40) [MSC v.1900 64 bit (AMD64)]
```

**Comment by PCManticore** [comment]
I can no longer reproduce this with `scipy==1.0.1` and
```
pylint 2.5.4
astroid 2.5.0
Python 3.6.5 (default, May  7 2018, 10:07:07)
[GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.42)]
```
